src/farmer.rs

```diff
@@ -246,20 +229,18 @@ impl Farmer {
     }
 
     // Stops farming, closes plots, and sends signal to the node
-    pub async fn close(mut self) -> anyhow::Result<()> {
-        let (stop_sender, stop_receiver) = futures::channel::oneshot::channel();
+    pub async fn close(self) {
+        let (stop_sender, stop_receiver) = oneshot::channel();
         if self
             .cmd_sender
             .send(FarmerCommand::Stop(stop_sender))
-            .await
             .is_err()
         {
-            return Err(anyhow::anyhow!("Noone is listening for our commands"));
+            return;
```

Comment:
if there is an error in line 237 here, I guess that is not an error anymore? Why did you abandon error handling for this function?

Reply:
I abandoned it because `Farmer` struct is essentially a smart pointer which handles process. It might reference to already exited farmer (for example, other task have already closed the farmer), so it is not an actual error to me.